images/go/image.yaml

```diff
@@ -13,11 +13,11 @@ versions:
       config: configs/1.19-dev.apko.yaml
       extractTagsFrom:
         package: go-1.19
-        prefix: dev-
         exclude:
           - 1
   - apko:
       config: configs/latest-dev.apko.yaml
       extractTagsFrom:
-        prefix: dev-
         package: go-1.20
+      tags:
+        - go-1.20-dev
```

Comment:
why have a tag with "go-" in the name? this is the go image. Are you looking for a tag suffix basically?

Reply:
good catch, ill remove the go-